### ISSUE TITLE
x-axis date-value in line-chart will be converted to date-string without time-part for series-tooltip

### ISSUE THREAD

**Issue by tobigit** [swimlane/ngx-charts#457]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here
```

**Current behavior**
I would like to create a special series tooltip with an own seriesTooltipTemplate. The tooltip should show the hovered time at the head of the tooltip, like "28.06.2017 13:15:00" (German Date Format). But the "name"-value in the tooltip model shows only the date-value as string without the time, like this: "28.6.2017"!

**Expected behavior**
The "name"-value of the tooltip-model should contain the raw date-value as it is. So i can do any date-conversion as i want.

**Reproduction of the problem**
Here is a plunker: https://plnkr.co/edit/n0N8GJfKrd7RHRNDmRo3?p=preview
When you open the console and hover the chart you can see the tooltip-model and the converted date-value without time-part:
![image](https://user-images.githubusercontent.com/1849231/27626541-bcde960a-5be8-11e7-910a-a77fcad6cadd.png)

**What is the motivation / use case for changing the behavior?**
Customize series tooltips with date values.

**Please tell us about your environment:**
Windows 7, IntelliJ, Angular-CLI

* **ngx-charts version:** 5.3.1

* **Angular version:** 4.0.1

* **Browser:** Chrome 59.0.3071.115

* **Language:** all

**Comment by tobigit** [comment]
When this feature is accepted the users can display the date and time value in the series-group-tooltip:
![image](https://user-images.githubusercontent.com/1849231/27955347-b227ea44-6314-11e7-854d-bda596dacbd0.png)

**Comment by marjan-georgiev** [comment]
Released in 6.1.0

**Comment by KambagiriswamyBo-Kore** [comment]
How to enable this feature ( Showing x axis date & time in tooltip )? I'm using version 10.0.0
